full-node/db/sov-db/src/ledger_db/mod.rs

```diff
@@ -211,6 +218,8 @@ impl LedgerDB {
             // The lock is released here
```

Comment:
Can't point this out (because its not in diff) but in line 208, there's a TODO for exposing a batch API, which we seem to implement now. Might want to remove/update that TODO.

Reply:
Good call. Removing!